src/engraving/libmscore/noteentry.cpp

```diff
@@ -387,8 +387,8 @@ void Score::putNote(const Position& p, bool replace)
     staff_idx_t staffIdx = track2staff(_is.track());
     if (m->isMeasureRepeatGroup(staffIdx)) {
         auto b = MessageBox::warning(trc("engraving", "Note input will remove measure repeat"),
-                                     trc("engraving", "There is a measure repeat here.")
-                                     + trc("engraving", "\nContinue with adding note and delete measure repeat?"));
+                                     trc("engraving", "There is a measure repeat here."
+                                                      "Continue with adding note and delete measure repeat?"));
```

Comment:
A space is missing at the end of the first line. Now the string wil be
```
"There is a measure repeat here.Continue with adding note and delete measure repeat?"
```
Also, how about changing the string to something like this:
```
"This measure contains a measure repeat. If you enter notes here, it will be deleted. Do you want to continue?"
```
Isn't that better/nicer English?

Reply:
yep, done